builtin/logical/pki/cert_util.go

```diff
@@ -361,6 +362,13 @@ func validateNames(req *logical.Request, names []string, role *roleEntry) string
 						break
 					}
 				}
+
+				if role.AllowGlobDomains &&
+					strings.Contains(currDomain, "*") &&
```

Comment:
Read back on the conversation on the PR and understood how not having this check would implicitly allow bare domain. Ignore my comment.